core/src/main/java/com/homeaway/streamingplatform/extensions/validation/EmptySchemaRegistrar.java

```diff
@@ -0,0 +1,47 @@
+/* Copyright (c) 2018 Expedia Group.
+ * All rights reserved.  http://www.homeaway.com
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+
+ *      http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.homeaway.streamingplatform.extensions.validation;
+
+import java.util.Map;
+
+import lombok.extern.slf4j.Slf4j;
+
+import com.homeaway.streamingplatform.exceptions.SchemaRegistrationException;
+import com.homeaway.streamingplatform.model.Stream;
+
+@SuppressWarnings("unused")
+@Slf4j
+public class EmptySchemaRegistrar implements SchemaRegistrar {
+
+    @Override
+    public boolean isSchemaValid(Stream stream) {
+        return false;
```

Comment:
I believe we will have a sub-module `confluent-schema-registry-validator` in this project, which has the concrete implementation for communicating with SchemaRegistry.

However, most other companies may not have SchemaRegistry setup, and might be using the EmptySchemaRegistrar implementation to make the app run.

In that case, do you think, `isSchemaValid` should always return true so that the app will function.